vnext/Microsoft.ReactNative/Views/Text/TextHitTestUtils.cpp

```diff
@@ -238,9 +238,14 @@ bool TextHitTestUtils::HitTest(const xaml::Documents::Run &run, const winrt::Poi
   // c. Skip step 4 if the first character is LTR
   // d. Skip step 5 if the last character is LTR
 
+  const auto parent = start.VisualParent();
+  if (!parent) {
+    return false;
+  }
+
   // Get the parent bounds
   const auto maxLeft = 0;
-  const auto maxRight = static_cast<float>(start.VisualParent().Width());
+  const auto maxRight = static_cast<float>(parent.Width());
```

Comment:
Width can be more (or less) than what it actually rendered, I think you want ActualWidth here instead.